.circleci/config.yml

```diff
@@ -25,9 +25,15 @@ commands:
               source .circleci/setup_env.sh
               conda install -y -c conda-forge git-lfs
               git lfs install --skip-repo --skip-smudge
-              git clone --recursive --depth=1 --shallow-submodules git@github.com:pytorch/benchmark.git torchbenchmark
-              (cd torchbenchmark && python install.py && touch env-v6.key)
+              rm -rf torchbenchmark
+              git clone --recursive git@github.com:pytorch/benchmark.git torchbenchmark
+              cd torchbenchmark
+              # Pin to specific version to avoid upstream breakages
+              git checkout 24b95f2f627bf07a61cefed653419389a7586357
```

Comment:
Nit: We should probably cut a semver tag / release in that repo, makes for a more coherent pin. It would then allow us to use 'git clone --branch'